src/smoke/processes.ts

```diff
@@ -1,34 +1,18 @@
 import { assert } from 'chai';
-
 import { SuiteMethod } from '../interfaces';
 import { getRunningSlackProcessesCount } from '../helpers/get-running-slack-processes';
 import { isWin } from '../utils/os';
 import { appState } from '../renderer/state';
-import { stopClientDriver, startClientDriver } from '../renderer/client-driver';
-
+import { startClientDriver } from '../renderer/client-driver';
+import { quit } from '../helpers/quit-app';
 export const test: SuiteMethod = async ({ it }) => {
   it('leaves no processes behind after closing', async () => {
-    // No need to be nice about it
-    await stopClientDriver();
-
-    // Wait until we have no processes
-    const runningSlackProcessesCount = await getRunningSlackProcessesCount(
-      appState
-    );
-    assert.equal(
-      runningSlackProcessesCount,
-      0,
-      `number of running processes is ${runningSlackProcessesCount}, expected 0.`
-    );
-
-    // Restart the app
+    const runningSlackProcessesCount = await quit();
```

Comment:
I think would still want to assert this equality here for the purposes of the test (even though the helper now also checks for the number of processes).
```suggestion
    await quit();
    const runningSlackProcessesCount = await getRunningSlackProcessesCount(	
      appState	
    );	
    assert.equal(	
      runningSlackProcessesCount,	
      0,	
      `number of running processes is ${runningSlackProcessesCount}, expected 0.`	
    );
```